### ISSUE TITLE
Prometheus - kubernetes-nodes-exporter endpoints down 

### ISSUE THREAD

**Issue by kisamoto** [openshift/openshift-ansible#7999]
#### Description

After clean installation of OCP 3.9 on RHEL 7, prometheus is unable to access the node exporters on any node other than the node it is deployed on. Instead it lists them all as `DOWN` with the error `Get http://{{ address }}:9100/metrics: dial tcp {{ address }}:9100: getsockopt: no route to host`

Possible firewall issue not opening port 9100 as `curl http://localhost:9100/metrics` responds as expected.

![image](https://user-images.githubusercontent.com/1744908/38877505-1aeca338-425f-11e8-932c-4e452dc225be.png)

##### Version

```
$ ansible --version
ansible 2.5.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/devops/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/share/venv/lib/python2.7/site-packages/ansible
  executable location = /usr/share/venv/bin/ansible
  python version = 2.7.5 (default, Feb 20 2018, 09:19:12) [GCC 4.8.5 20150623 (Red Hat 4.8.5-28)]

$ rpm -q openshift-ansible
openshift-ansible-3.9.14-1.git.3.c62bc34.el7.noarch
```

##### Steps To Reproduce
1. Install Openshift enterprise 3.9 with prometheus enabled
2. Access `https://prometheus-url/targets`

##### Expected Results

Show `kubernetes-nodes-exporter (14/14 up)` and `kubernetes-service-endpoints (15/15 up)`

##### Observed Results

Seeing `kubernetes-nodes-exporter (1/14 up)` and `kubernetes-service-endpoints (0/15 up)` with error `Get http://{{ address }}:9100/metrics: dial tcp {{ address }}:9100: getsockopt: no route to host` however able to `curl http://localhost:9100/metrics` from node with node-exporter installed and get expected result.

##### Additional Information

- Can `curl` for metrics on node-exporter localhost
- Using the `redhat/openshift-ovs-multitenant` plugin
- Possible firewall issue
```
$ sudo firewall-cmd --list-all-zones

[. . .]

internal
  target: default
  icmp-block-inversion: no
  interfaces:
  sources:
  services: ssh mdns samba-client dhcpv6-client
  ports:
  protocols:
  masquerade: no
  forward-ports:
  source-ports:
  icmp-blocks:
  rich rules:

public (active)
  target: default
  icmp-block-inversion: no
  interfaces: eth0
  sources:
  services: ssh dhcpv6-client
  ports: 2379/tcp 2380/tcp 8443/tcp 8444/tcp 8053/tcp 8053/udp 9090/tcp 10250/tcp 80/tcp 443/tcp 4789/udp
  protocols:
  masquerade: no
  forward-ports:
  source-ports:
  icmp-blocks:
  rich rules:

[. . .]

```

**Comment by sdodson** [comment]
https://bugzilla.redhat.com/show_bug.cgi?id=1563888 on this too

**Comment by simonpasquier** [comment]
This should be fixed by #9072.
